python/DEVELOPER.md

```diff
@@ -16,7 +16,7 @@ Software Dependencies
 -   git
 -   GCC
 -   pkg-config
--   protoc (protobuf compiler) >= v3.20.0
+-   protoc (protobuf compiler) >= v26.1
```

Comment:
We don't want to impose strict restrictions on other wrappers only since Java requires it - so lets keep it only for Java